cmd/debos/debos.go

```diff
@@ -317,7 +316,8 @@ func main() {
 			}
 		}
 
-		m.SetQuiet(options.QuietFakeMachine)
+		// Silence extra output from fakemachine unless the --verbose flag was passed.
```

Comment:
can you please squash the two commits & use the top commit as the commit message ?

Reply:
Sure! Done.